app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -51,8 +52,10 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
         super.onViewCreated(view, savedInstanceState)
 
         adapter.addLoadStateListener { loadState ->
-            binding.transactions.isVisible = loadState.refresh is LoadState.NotLoading
-            binding.progress.isVisible = loadState.refresh is LoadState.Loading
+            if (lifecycle.currentState.isAtLeast(Lifecycle.State.STARTED)) {
```

Comment:
this should be added also to adapter.addDataRefreshListener

Reply:
Done in 326ae7b0ef245854436e0dba1d1331b46e414f57